apps/modernization-api/src/main/resources/graphql/coded-values.graphqls

```diff
@@ -150,6 +161,6 @@ extend type Query {
     findAllPhoneAndEmailUse(page: Page): PhoneAndEmailUseResults!
     findAllIdentificationTypes(page: Page): IdentificationTypesResults!
     findAllAssigningAuthorities(page: Page): AssigningAuthorResults!
-    findAllStateCountyCodeValues(page: Page): StateCountyCodeValueResults!
+    findAllStateCountyCodeValues(page: Page): [StateCountyCodeValue!]!
```

Comment:
Query definition doesn't match the QueryMapping method.  Is a Page needed here if the results are a list?